spec/controllers/assets_controller_spec.rb

```diff
@@ -254,25 +254,11 @@
     describe "cache headers" do
       let(:asset) { FactoryBot.create(:asset) }
 
-      it "sets the cache-control headers to 0 for an unscanned asset" do
+      it "sets the cache-control headers to 0" do
```

Comment:
s/headers/header

Reply:
Thanks. I've addressed this in [a new commit](https://github.com/alphagov/asset-manager/pull/458/commits/9e92fdebccb9c56f384e50f3b3d21a6525e7568a), because this wasn't something that I introduced.